assessment_store/api/routes/flag_routes.py

```diff
@@ -69,20 +73,24 @@ def create_team_dict(team_name):
     return team_flag_stats
 
 
-def get_flag(flag_id: str):
+@assessment_flag_bp.get("/flag_data")
+def get_flag():
+    flag_id = request.args["flag_id"]
```

Comment:
As a reminder to myself, previous checking done by connexion (assuming it makes sure there's a flag id query string here) now _won't be done_ -- this should be fine if we're moving away from the stores exposing endpoints